Dockerfile

```diff
@@ -1,6 +1,8 @@
-FROM ubuntu:14.04
+FROM progrium/busybox
```

Comment:
Note, if bad things is breakages that's only on rebuild, the same goes for security issues only when we rebuild, that we have to trust...

But yeah we might want to keep an eye for better base image. IMO it's worth freeing up 200mb on every single worker, especially since docker likes to keep things in memory.